loopr-ios/Setting/SettingManageWallet/ChangeWalletName/SettingChangeWalletNameViewController.swift

```diff
@@ -8,22 +8,69 @@
 
 import UIKit
 
-class SettingChangeWalletNameViewController: UIViewController {
+class SettingChangeWalletNameViewController: UIViewController, UITextFieldDelegate {
 
     var appWallet: AppWallet!
+    var nameTextField: UITextField = UITextField()
+    var nameFieldUnderLine: UIView = UIView()
+    var saveButton: UIButton = UIButton()
     
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        // Setup UI in the scroll view
+        let screensize: CGRect = UIScreen.main.bounds
+        let screenWidth = screensize.width
+        // let screenHeight = screensize.height
+        
+        let originY: CGFloat = 30
+        let padding: CGFloat = 15
+        
         // Do any additional setup after loading the view.
         self.navigationItem.title = NSLocalizedString("Change Wallet Name", comment: "")
         setBackButton()
-
+        
+        nameTextField.delegate = self
+        nameTextField.tag = 0
+        nameTextField.inputView = UIView()
+        nameTextField.font = FontConfigManager.shared.getLabelFont()
+        nameTextField.theme_tintColor = GlobalPicker.textColor
+        nameTextField.placeholder = "Enter your wallet name"
+        nameTextField.contentMode = UIViewContentMode.bottom
+        nameTextField.frame = CGRect(x: padding, y: originY, width: screenWidth-padding*2-80, height: 40)
+        self.view.addSubview(nameTextField)
+        
+        nameFieldUnderLine.frame = CGRect(x: padding, y: nameTextField.frame.maxY, width: screenWidth - padding * 2, height: 1)
+        nameFieldUnderLine.backgroundColor = UIColor.black
+        self.view.addSubview(nameFieldUnderLine)
+        
+        saveButton.setTitleColor(UIColor.white, for: .normal)
```

Comment:
Please use ```saveButton.setupRoundBlack()```

Reply:
updated.

Reply:
Great. The only thing left is project.pbxproj should be reverse.